superset/assets/src/visualizations/deckgl/layers/common.jsx

```diff
@@ -71,7 +71,7 @@ export function commonLayerProps(formData, setTooltip, onSelect) {
         setTooltip({
           content: tooltipContentGenerator(o),
           x: o.x,
-          y: o.y,
+          y: o.y + 30,
```

Comment:
What does the `+30` do? Reposition the tooltip to a better place?

Reply:
Yeah, for some reason the tooltip is offset vertically, and this fixes it.

Reply:
👍